int64 starting_timestamp
int64 ending_timestamp
string classification